x-pack/solutions/security/packages/kbn-cloud-security-posture/public/src/constants/component_constants.ts

```diff
@@ -7,8 +7,8 @@
 import { euiThemeVars } from '@kbn/ui-theme';
 
 export const statusColors = {
-  passed: euiThemeVars.euiColorSuccess,
-  failed: euiThemeVars.euiColorVis9,
+  passed: '#00bfb3',
+  failed: '#e7664c',
```

Comment:
we should probably name those, and maybe ask design to select colors which are part of eui